packages/components/components/elvis-timepicker/src/react/elviaTimepicker.types.tsx

```diff
@@ -2,6 +2,8 @@ import { BaseProps, type FormFieldSizes, HasError, HasValue } from '@elvia/elvis
 
 import { MinuteInterval } from './publicApi.public';
 
+export type ChangeType = 'hour' | 'minute' | 'second';
```

Comment:
Is there a reason this was created? It seems like we have this type in `elvis-timepicker/src/react/publicApi.public.ts` that we should reuse instead

Reply:
There was an error related to it not being exported correctly, and when I checked the file it was complaining about I noticed what it was asking for didnt exist within the file. I then created a new `export type` called `ChangeType` here not knowing this already existed on `publicApi.public.ts`. This has now been resolved.